tox.ini

```diff
@@ -65,7 +65,7 @@ commands =
     yarn
     make dist
     # Test whl installation
-    pip install {toxinidir}/dist/kolibri_explore_plugin-0.0.1-py2.py3-none-any.whl
+    pip install {toxinidir}/dist/kolibri_explore_plugin-*.whl
```

Comment:
This is not working, I think it's because the tox tooling that sees that like a filename and doesn't apply the glob. Maybe we should add the version here like a variable

Reply:
Yeah. I don't know about tox but I will check how to do that.

Reply:
Maybe this is interesting: https://github.com/tox-dev/tox/issues/1571

Reply:
If it's too complex we can just remove this line from tests, it's something not required to test. With the `make dist` is enough.

Reply:
Indeed. I updated the PR disabling the command and referencing the ticket in tox.